src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,67 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (newPage: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const linksAmount = Math.ceil(total / perPage);
+  const linkNumbers = getNumbers(1, linksAmount);
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
+          onClick={() => onPageChange(currentPage - 1)}
```

Comment:
You should prevent the default action of the anchor element to avoid changing the URL fragment when clicking the previous page link. You can do this by calling `event.preventDefault()` in the `onClick` handler.